django/views/debug.py

```diff
@@ -893,6 +893,28 @@ def default_urlconf(request):
   <h2>Request information</h2>
 
 {% if request %}
+  <h3 id="user-info">USER</h3>
+  {% if request.user %}
+    <table class="req">
+      <thead>
+        <tr>
+          <th>Variable</th>
+          <th>Value</th>
+        </tr>
+      </thead>
+      <tbody>
+        {% for var in request.user.items %}
```

Comment:
Is this meant to do anything? `AttributeError: 'User' object has no attribute 'items'`
I don't think it's necessary. Some fields on the user model could be sensitive too.
A mention in the 1.10 release notes is appropriate too.